requirements.txt

```diff
@@ -1,259 +1,16 @@
-    --hash=sha256:bfdf460b1736c775f2ba9f6a92bca30bc2095067b8a9d77876d1fad6cc3b4a43 \
-    --hash=sha256:c8098ddcc2a85b61647b2590f825f3db38891662cfc2fc776415143f599bb859 \
-    --hash=sha256:d2b04aac4d386b172d5b9692e2d2da8de7bfb6c387fa4f801fbf6fb2e6ba4673 \
-    --hash=sha256:d858aa552c999bc8a8d57426ed01e40bef403cd8ccdd0fc5f6f04a00414cac2a \
-    --hash=sha256:f003ed9ad21d6a4713f0a9b5a7a0a79e08dd0f221aff4525a2be4c346ee60aab \
-    --hash=sha256:f22ac1c3cac4dbc50079e965eba2c1058622631e526bd9afd45fedd49ba781fa \
-    --hash=sha256:faca3bdcf85b2fc05d06ff3fbc1f83e1391b3e724afa3feba7d13eeab355484c \
-    --hash=sha256:fca0e3a251908a499833aa292323f32437106001d436eca0e6e7833256674585 \
-    --hash=sha256:fd1592b3fdf65fff2ad0004b5e363300ef59ced41c2e6b3a99d4089fa8c5435d \
-    --hash=sha256:fd66fc5d0da6d9815ba2cebeb4205f95818ff4b79c3ebe268e75d961704af52f
-tenacity==8.2.2 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0" \
-    --hash=sha256:2f277afb21b851637e8f52e6a613ff08734c347dc19ade928e519d7d2d8569b0 \
-    --hash=sha256:43af037822bd0029025877f3b2d97cc4d7bb0c2991000a3d59d71517c5c969e0
-websocket-client==1.5.2 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0" \
-    --hash=sha256:c7d67c13b928645f259d9b847ab5b57fd2d127213ca41ebd880de1f553b7c23b \
-    --hash=sha256:f8c64e28cd700e7ba1f04350d66422b6833b82a796b525a51e740b8cc8dab4b1
-zipp==3.15.0 ; python_full_version >= "3.8.10" and python_version < "3.9" \
-    --hash=sha256:112929ad649da941c23de50f356a2b5570c954b65150642bccdd66bf194d224b \
-    --hash=sha256:48904fc76a60e542af151aded95726c1a5c34ed43ab4134b597665c86d7ad556
+attrs==23.1.0 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+cffi==1.15.1 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+cryptography==40.0.2 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+dnspython==2.3.0 ; python_full_version >= "3.8.10" and python_version < "4.0"
+importlib-resources==5.12.0 ; python_full_version >= "3.8.10" and python_version < "3.9"
+jsonschema==4.17.3 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+ops==2.4.1 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+pkgutil-resolve-name==1.3.10 ; python_full_version >= "3.8.10" and python_version < "3.9"
+pluggy==1.2.0 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+pycparser==2.21 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+pymongo==4.3.3 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+pyrsistent==0.19.3 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+pyyaml==6.0.1 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+tenacity==8.2.2 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+websocket-client==1.5.2 ; python_full_version >= "3.8.10" and python_full_version < "4.0.0"
+zipp==3.15.0 ; python_full_version >= "3.8.10" and python_version < "3.9"
```

Comment:
I thought we were going to pin the charmcraft version? Or did this not work?

Reply:
This was commited by mistake here. Let me revet change

Reply:
+1